_boards_cm/xerxes-pi.md

```diff
@@ -6,8 +6,28 @@ status: prototype
 picture: "/images/board-xerxes-pi.jpeg"
 github_issue: "https://github.com/geerlingguy/raspberry-pi-pcie-devices/issues/753"
 link: https://www.kickstarter.com/projects/1907647187/small-board-big-possibilities-xerxes-pi
-videos: []
+videos: 
+  - https://youtu.be/_bxnmqnCeCg
+  - https://youtu.be/4dGTC-YSq1g
+  - https://youtu.be/nelqV0-S2C0
+  - https://youtu.be/4I_BMbAXmSg
 ---
 The Xerxes Pi is a Compute Module blade designed for 1U racks designed by Rapid Analysis.
 
-It was launched on Kickstarter, and is estimated to ship in late 2025. The board includes USB-C power input, a micro USB port, and a 1 Gbps Ethernet jack. Optional PoE support can be added, and there's an M.2 E-key (up to 2260 size) and microSD card slot on the bottom.
+It was launched on [Kickstarter](https://www.kickstarter.com/projects/1907647187/small-board-big-possibilities-xerxes-pi), and is estimated to ship in late 2025. The board includes:
+
+- USB-C on-the-go port for power and data
+- 1 Gbps Ethernet jack
+- M.2 M-key (up to 2280 size)
+- microSD card slot on the bottom
+- I2C qwiic connector
+- SH-04 PWM Fan connector
+- JTAG UART Debug connector
+
+![Image](https://rapidanalysis.com/xerxes/xerxes_prototype.png)
```

Comment:
Can you just leave in the single photo that's referenced in the metadata? I typically like to have these pages link to the official site for more photos and details beyond an initial overview. If there's too much information/photos on the PiPCIe site for any particular products, then it becomes a bit of a maintenance burden.